cmdx/shell.go

```diff
@@ -11,8 +11,12 @@ import (
 // bash, zsh, fish, and powershell. It should be added on the root
 // command and can be used as `completion bash` or `completion zsh`.
 func SetCompletionCmd(exec string) *cobra.Command {
+	var execs []interface{}
```

Comment:
@mabdh can you explain the need of this?

Reply:
The command does not render properly if we run `go run . completion -h`, `heredoc.Docf` string interpolation requires the same number of `%s` and args itself. What we have is, the argument is only exec, when running `go run . completion -h`, it generates this
```bash
Bash:

  $ source <(guardian completion bash)

  # To load completions for each session, execute once:
  # Linux:
  $ %!s(MISSING) completion bash > /etc/bash_completion.d/%!s(MISSING)
  # macOS:
  $ %!s(MISSING) completion bash > /usr/local/etc/bash_completion.d/%!s(MISSING)

Zsh:

  # If shell completion is not already enabled in your environment,
  # you will need to enable it.  You can execute the following once:

  $ echo "autoload -U compinit; compinit" >> ~/.zshrc

  # To load completions for each session, execute once:
  $ %!s(MISSING) completion zsh > "${fpath[1]}/_yourprogram"

  # You will need to start a new shell for this setup to take effect.

fish:

  $ %!s(MISSING) completion fish | source

  # To load completions for each session, execute once:
  $ %!s(MISSING) completion fish > ~/.config/fish/completions/%!s(MISSING).fish

PowerShell:

  PS> %!s(MISSING) completion powershell | Out-String | Invoke-Expression

  # To load completions for every new session, run:
  PS> %!s(MISSING) completion powershell > %!s(MISSING).ps1
  # and source this file from your PowerShell profile.

```